ConsoleAppHomeTask/Program.cs

```diff
@@ -6,11 +6,61 @@
 
 namespace ConsoleAppHomeTask
 {
+    enum TransactionType { Buy, Sell }
     class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello world!");
+            //Console.WriteLine("Hello world!");
```

Comment:
Please remove unnecessary comments and looks like there formatting issues at the end of this file. Try use Edit -> Advanced -> Format Current Document in VS